design.md

```diff
@@ -2,12 +2,41 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We assume that users will have familiarity with the Picobot world and basic computer science concepts. That is, while
+we expect the users will have understanding of the Picobot simulation and state machines, we do not expect the 
+users will have knowledge of advanced programming and computer science. Ideally, students with a CS5 level of
+eductation should be able to work with this language design.
+
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+While our language design is somewhat similar to the original picobot language, we feel our language design improves upon
+lacking aspects of the original design. Since we saw a majority of good "readable" picobot code groups the same state
+lines together, we require all states to be grouped together under a keyword. This both saves time coding (by preventing
+duplicate coding) and enforcing good organization style. Additionally, by using unicode characters, we feel it is more
+intuitive and visually understandable. Furthermore, we chose to remove the ordering requirement from the arrows. Even 
+though the original NEWS ordering was simple enough (due to the acronymy), removing this forced restriction improves
+expression. Additionally, we list the Open arrows second since it makes it closer to the move direction arrow. This makes
+sense since if the Move Direction needs to be in the Open set to ensure no errors.
+
+
```

Comment:
I do like the unicode --> definitely more intuitive and readable. IMO, it also alleviates confusion you were concerned about below (with confusion based on blocked/unblocked being done separately). However, unicode characters are not on everyone's keyboard; could be annoying to insert symbol every time you need to use one.